internal/app/cache/gcp_cache.go

```diff
@@ -0,0 +1,50 @@
+package cache
+
+import (
+	"context"
+
+	discovery "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	gcp "github.com/envoyproxy/go-control-plane/pkg/cache"
+	orchestrator "github.com/envoyproxy/xds-relay/internal/app/orchestrator"
```

Comment:
Can drop the package rename if it's the same as the last import.
```suggestion
	"github.com/envoyproxy/xds-relay/internal/app/orchestrator"
```